src/clock/mumeclockwidget.cpp

```diff
@@ -160,14 +160,14 @@ void MumeClockWidget::slot_updateLabel()
     } else
         timeLabel->setText(m_clock->toCountdown(moment));
 
-    const MumeMoonVisibilityEnum moonVisibility = moment.toMoonVisibility();
+    const MumeMoonVisibilityEnum moonVisibility = moment.moonVisibility();
     if (moonVisibility != m_lastVisibility || updateMoonStyleSheet) {
         m_lastVisibility = moonVisibility;
-        const QString moonStyleSheet = (moonVisibility == MumeMoonVisibilityEnum::HIDDEN)
+        const QString moonStyleSheet = (moonVisibility <= MumeMoonVisibilityEnum::INVISIBLE)
```

Comment:
This <= feels odd to me from a readability standpoint. Is there a simpler way to do this?

Reply:
I'll just do an or instead :-)